services/web/client/source/class/osparc/component/metadata/ServicesInStudy.js

```diff
@@ -27,38 +27,69 @@ qx.Class.define("osparc.component.metadata.ServicesInStudy", {
   construct: function(studyData) {
     this.base(arguments);
 
-    this._setLayout(new qx.ui.layout.Grid(10));
+    const grid = new qx.ui.layout.Grid(10);
+    grid.setColumnFlex(this.self().gridPos.name, 1);
```

Comment:
``this.self``?? sounds like two languages together to identify current instance.  you cannot just ``this.gridPos``?

Reply:
That's the static methods/variables calling way. I think that what you suggest only works for static classes.